script/options.js

```diff
@@ -76,7 +77,8 @@ function applyEnvironmentOverrides(options) {
 function applyBrandConsolidationOverrides(options) {
   Object.assign(options, {
     contentRoot: '../va-gov',
-    collections: require('./collections/brand-consolidation.json')
+    collections: require('./collections/brand-consolidation.json'),
+    redirects: require('../config/vagovRedirects.json')
```

Comment:
Since the redirects are set up in Metalsmith, can we store that JSON file in the `script` directory?

Reply:
Sure